File: B/Ruby/b008.rb
max = 0
incom = []
s = gets.chomp.split(" ")
N = s[0].to_i
M = s[1].to_i
if M>0 && N>0 then
    M.times {
        e = gets.chomp.split(" ")
        sum = 0
        e.each { |n|
            sum += n.to_i
        }
        incom << sum
    }
    incom.each { |n|
        if n > 0 then max += n end
    }
end
puts max

File: B/Swift/b008.swift
extension String: CollectionType {}

let tmp = readLine()!.split(" ")
let N = Int(tmp[0])!
let M = Int(tmp[1])!
var max = 0
var incom = Array(count: M, repeatedValue: 0)

if M>0 && N>0 {
  for i in 0 ..< M {
    var sum = 0
    var tmp: [Int] = readLine()!.split(" ").map { Int($0)! }
    for j in 0 ..< N {
      sum += tmp[j]
    }
    incom[i] = sum
  }
  for i in 0 ..< M {
    var n = incom[i]
    if n>0 {
      max += n
    }
  }
}
print(max)
